src/addr.rs

```diff
@@ -386,13 +386,12 @@ impl PhysAddr {
     ///
     /// This function panics if a bit in the range 52 to 64 is set.
     #[inline]
-    pub fn new(addr: u64) -> PhysAddr {
-        assert_eq!(
-            addr.get_bits(52..64),
-            0,
-            "physical addresses must not have any bits in the range 52 to 64 set"
-        );
-        PhysAddr(addr)
+    pub const fn new(addr: u64) -> Self {
+        // TODO: Replace with .ok().expect(msg) when that works on stable.
+        match Self::try_new(addr) {
+            Ok(p) => p,
+            Err(_) => panic!("physical addresses must not have any bits in the range 52 to 64 set"),
+        }
     }
```

Comment:
The `use bit_field::BitField;` import at the top of the file is now dead code and can be removed.
(Github won't let me comment on a line that wasn't changed, so I'm putting this here)

Reply:
Looks like it has to be conditionally imported if `#[cfg(feature = "step_trait")]`

Should be fixed now.